docs/scripts/regen_kind_tags.py

```diff
@@ -11,7 +11,7 @@
         "dag",
         "task",
     ],
-    ["table", "view", "source", "seed", "file", "dashboard", "notebook"],
+    ["table", "view", "semanticmodel", "source", "seed", "file", "dashboard", "report", "notebook"],
```

Comment:
oh whoa didn't realize this script exists... I manually added a kind tag yesterday. now i know.

Reply:
yea, you can use `make rebuild-kind-tags-page`